actions/src/main/res/color/button_color_change.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="true" android:color="@color/shutter_photo_tint" />
```

Comment:
See my below comment on the style but fyi you should mostly try to use the `?attr/` referencing theme resources.
Try to even avoid using theme colors directly if you can. E.g. using `alfresco_blue_700` inside `shutter_photo_tint` was an exception because it here but I had a different though which I would go now and replace with `?attr/colorPrimary`

Reply:
Please change it for shutter and I'll check it and implement accordingly

Reply:
Surprisingly I tried to change it, but it crashes in the objectAnimator. Seems that doing valueFrom/To while referencing a color which references a theme attribute is too much for it. Would probably work if we code the animator but that's too much work for little benefit.

Reply:
leave it as it's for now. if there is some major change then will consider it.